zynqmp.mk

```diff
@@ -54,10 +68,15 @@ petalinux-config:
 	$(call set_optee_version,$(OPTEE_VER),$(PRJ_PATH)/project-spec/meta-user/recipes-apps/optee-test/optee-test.bbappend)
 	$(call set_optee_version,$(OPTEE_VER),$(PRJ_PATH)/project-spec/meta-user/recipes-apps/optee-client/optee-client.bbappend)
 	$(call set_optee_version,$(OPTEE_VER),$(PRJ_PATH)/project-spec/meta-user/recipes-bsp/optee-os/optee-os.bbappend)
-	@cd $(PRJ_PATH) && petalinux-config --oldconfig
+	@petalinux-config -p $(PRJ_PATH) --oldconfig
```

Comment:
Curious to know, are these executables (... config,build etc) setup also when running `settings.sh`?

Reply:
yes, `settings.sh` is used to setup everything related to petalinux (variable, paths to binaries, etc.)